core/src/main/java/com/linecorp/armeria/server/VirtualHostAnnotatedServiceBindingBuilder.java

```diff
@@ -275,11 +277,24 @@ public VirtualHostAnnotatedServiceBindingBuilder decorator(
      * @return {@link VirtualHostBuilder} to continue building {@link VirtualHost}
      */
     public VirtualHostBuilder build(Object service) {
+        this.service = service;
+        virtualHostBuilder.addAnnotatedServiceBindingBuilder(this);
+        return virtualHostBuilder;
+    }
+
+    /**
+     * Apples the {@link ServiceConfigBuilder} created with the configured
+     * {@link AnnotatedHttpServiceExtensions} to the {@link VirtualHostBuilder}.
+     *
+     * @param extensions the {@link AnnotatedHttpServiceExtensions} at the virtual host level.
+     */
+    VirtualHostBuilder applyToServiceConfigBuilder(AnnotatedHttpServiceExtensions extensions) {
```

Comment:
This method does not have to return `VritualHostBuilder` as we do in `AnnotatedServiceBindingBuilder`.